_posts/2025-10-06-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,89 @@
+
+### Key Features
+* Native Prometheus support: The reporter exposes metrics in the Prometheus format through an HTTP endpoint, without relying on JMX.
+* Configurable Metrics Collection: Allows users to specify which metrics should be exposed using a flexible allowlist.
+
+#### Deploying Metrics Reporter
+The first step is to include the Strimzi Metrics Reporter in your `Kafka` custom resource.
+To do this, add the following:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+  # ...
+    metricsConfig:
+      type: strimziMetricsReporter
+  # ...  
+```
+
+By adding `type: strimziMetricsReporter` to the `metricsConfig` section of your `Kafka` custom resource, Strimzi will export a sensible set of default metrics.
+However, you can add you own custom values by filtering the metrics by name. 
+This is achieved by adding another field called `values`, and within that field, adding `allowList`.
+Then you can specify which metrics you want to collect, where each entry is used to filter allowed metrics.
+You can put individual metrics names or use a regex to include a group of metrics and avoid having a long list.
+For example:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+  # ...
+    metricsConfig:
+      type: strimziMetricsReporter
+      values:
+        allowList:
+          - "kafka_log.*"
+          - "kafka_network.*"
+  # ...  
+```
+
+If you are adding Strimzi Metrics Reporter to an existing cluster, then you will see all your Kafka broker and Controller pods roll so that the Strimzi Metrics Reporter config is set in your `Kafka` resource.
+If you would like to deploy your Kafka cluster with the Strimzi Metrics Reporter enabled from the start, you can use the examples in our [Strimzi Metrics Reporter Examples folder](https://github.com/strimzi/strimzi-kafka-operator/tree/0.48.0/examples/metrics/strimzi-metrics-reporter) by running the following command:
+
+```bash
+$ kubectl apply -f https://strimzi.io/examples/0.48.0/metrics/strimzi-metrics-reporter/kafka-metrics.yaml -n myproject
+```
+
+We also provide some [Grafana Dashboards](https://github.com/strimzi/strimzi-kafka-operator/tree/0.48.0/examples/metrics/strimzi-metrics-reporter/grafana-dashboards) to help you get started with visualizing your Kafka metrics in the Prometheus format.
+Strimzi Metrics Reporter can also be used with Strimzi Kafka Bridge, Kafka Connect and Kafka MirrorMaker 2, and we provide [examples](https://github.com/strimzi/strimzi-kafka-operator/tree/0.48.0/examples/metrics/strimzi-metrics-reporter) for these too.
+
+
+### Conclusion
+We have shown you how to deploy the Strimzi Metrics Reporter in your Kafka cluster and given some reasons why you might want to use it as an alternative to other monitoring tools.
```

Comment:
Again as it's still in early access, I'm not sure if we want to recommend it or simply ask for testing and feedback?